src/python/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+find_package(Python COMPONENTS Interpreter Development)
+find_package(pybind11 CONFIG)
+
+pybind11_add_module(pyf3d F3DPythonBindings.cxx)
+
+target_compile_features(pyf3d PRIVATE cxx_std_17)
```

Comment:
I woudl have expected cxx11 here, why is it needed ?

Reply:
cxx14 is needed for `py::overload_cast`, should I downgrade it to cxx14?

Reply:
no, I was afraid that f3d forced you to use cxx17 for some reason. this is fine.